Yoast/wordpress-seo
# 14302 remove floating save button

## Summary


This PR can be summarized in the following changelog entry:

* Removes the floating bar with the save changes button

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* On pages like general/features are now two save changes buttons, you can see it briefly change when scrolling down.
This PR removes one of them. When scrolling you should now see only one button. 

## UI changes

## Documentation

## Quality assurance


Fixes #14302
